Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -464,6 +464,7 @@ class BrowserViewController: UIViewController,
         ensureMainThread { [weak self] in
             guard let self else { return }
 
```

Comment:
I think we should keep the state in variable and update it when this method is called.